cmake/FindPMIx.cmake

```diff
@@ -3,13 +3,14 @@ pkg_check_modules(PC_PMIX QUIET pmix)
 
 find_path(PMIX_INCLUDE_DIR pmix.h
     HINTS
-        ${PMIX_ROOT}  ENV PMIX_ROOT
-        ${PMIX_DIR}   ENV PMIX_DIR
+    ${PMIX_ROOT}  ENV PMIX_ROOT
+    ${PMIX_DIR}   ENV PMIX_DIR
     PATH_SUFFIXES include)
 
-find_library(PMIX_LIBRARY NAMES pmix
+find_library(PMIX_LIBRARY HINT ${PMIX_DIR} NAMES pmix
```

Comment:
For symmetry with UCX - this actually lets cmake find the library provided by a module, if the module specifies `PMIX_DIR`

Reply:
sure